### ISSUE TITLE
Bug in Sample Code in chapter "A Basic Component"

### ISSUE THREAD

**Issue by liangcorp** [leptos-rs/book#54]
There is a bug in the first sample code in chapter "A Basic Component". `set_count(3)` should be `set_count.set(3)`.
```
#[component]
fn App() -> impl IntoView {
    let (count, set_count) = create_signal(0);

    view! {
        <button
            on:click=move |_| {
                set_count(3);
            }
        >
            "Click me: "


            {move || count.get()}
        </button>
    }
}
```
![Screenshot from 2024-01-29 01-17-57](https://github.com/leptos-rs/book/assets/2737157/db585d58-97a0-4778-9d70-a136c6e89990)

**Comment by liangcorp** [comment]
Understandable. I would suggest to use stable as documentation base after leptos reaches 1.0.

**Comment by gbj** [comment]
Closed as of 0.7 book update.
